core/src/test/scala/org/apache/spark/shuffle/hash/HashShuffleManagerSuite.scala

```diff
@@ -46,6 +46,7 @@ class HashShuffleManagerSuite extends SparkFunSuite with LocalSparkContext {
     // an object is written. So if the codepaths assume writeObject is end of data, this should
     // flush those bugs out. This was common bug in ExternalAppendOnlyMap, etc.
     conf.set("spark.serializer.objectStreamReset", "1")
+    conf.set("spark.shuffle.consolidateFiles", "true")
```

Comment:
@aarondav Do you remember why we did not enable consolidation in this test? Since the title is "consolidated shuffle can write to shuffle group without messing existing offsets/lengths"  (introduced in #1678)